### ISSUE TITLE
Possibly wrong groupPackage

### ISSUE THREAD

**Issue by codepitbull** [vert-x3/vertx-proton#65]
I stumbled over the following issue:
When using `type.module.translatePackageName('scala')` for vertx-proton I get **io.vertx.proton.scala** instead of **io.vertx.scala.proton** the reason lies in the groupPackage definition in package-info:
```
ModuleGen(name = "vertx-proton", groupPackage = "io.vertx.proton")
```
This differs from what all other modules use:
```
ModuleGen(name = "vertx-web-client", groupPackage = "io.vertx")
```

I guess the latter version is correct, or are there other reasons to include the module name in the groupPackage?

**Comment by vietj** [comment]
you are right, we should fix it in 3.6 I think

**Comment by gemmellr** [comment]
I have updated it as described.

I'd guess that I c&p the original from vertx-core and and changed it thinking that the groupPackage was the package of the module rather than always just "io.vertx".
